### DISCUSSION TITLE
SRX Stable - Mounting Type

### DISCUSSION THREAD

**Discussion by BigChinoDon** [FrSkyRC/ETHOS-Feedback-Community#4042] [Q&A]
Question regarding the 4 choices in "Mounting Type".  If the receiver is upside down, which mounting type should be selected?  (Horizontal or Horizontal reversed)  I assumed reversed meant the pins to the rear (TW SR8 manual) are intended to be towards the front.  However, some others online thinks it means inverted, AKA upside down.

Thanks,
Don

**Comment by Boebe62** [comment]
I just put an Archer SR10+ in a plane upside down. That’s horizontal reversed for the SR10+. 
Hope thus helps.
